Prefix/InfixToPrefixConverter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Project2_Group_17.Prefix
+{
```

Comment:
I feel like he in particular, and I kind of agree, doesn't really like the .FolderName namespace.
Maybe we should just keep all in the parent namespace, do you agree?

Reply:
Yeah overall it doesn't matter what kind of name spaces we store it in, I have handed in stuff to him using .folderName namespaces and he was indifferent. But given that everything is currently in a more generic namespace, I updated it to replicate that